app/types/types/lang.type.ts

```diff
@@ -0,0 +1,3 @@
+export interface LangType {
```

Comment:
It's not `LangType` it's a type of url params
```suggestion
export interface ParamsWithLanguage{
```